sdk/translation/Azure.AI.Translation.Document/tests/GetAllTranslationsFilterTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Translation.Document.Tests
+{
+    public partial class DocumentTranslationClientLiveTests
+    {
+        public async Task<List<string>> CreateTranslationJobsAsync(DocumentTranslationClient client, int jobsCount = 1, int docsPerJob = 1, DocumentTranslationStatus jobTerminalStatus = default)
+        {
+            // create source container
+            if (jobTerminalStatus == DocumentTranslationStatus.Cancelled)
+            {
+                docsPerJob = 20; // in order to avoid job completing before cancelling
+            }
+            var testDocs = CreateDummyTestDocuments(count: docsPerJob);
+            var sourceContainer = await CreateSourceContainerAsync(testDocs);
+
+            // create a translation job
+            var resultIds = new List<string>();
+            for (int i = 1; i <= jobsCount; i++)
+            {
+                var targetContainer = await CreateTargetContainerAsync();
+                var input = new DocumentTranslationInput(sourceContainer, targetContainer, "fr");
+                var translationOp = await client.StartTranslationAsync(input);
+                resultIds.Add(translationOp.Id);
+                if (jobTerminalStatus == DocumentTranslationStatus.Succeeded)
+                {
+                    await translationOp.WaitForCompletionAsync();
+                }
+                else if (jobTerminalStatus == DocumentTranslationStatus.Cancelled)
+                {
+                    await translationOp.CancelAsync(default);
+                    Thread.Sleep(3000); // wait for cancel status to propagate!
```

Comment:
do we need to do this per job? or just at the end of the function before returning the `resultIds`

Reply:
actually for each job in order to wait for 'cancel' status to propagate